modules/linear_boltzmann_solvers/c_discrete_ordinates_adjoint_solver/lbs_adj_solver.cc

```diff
@@ -38,33 +38,31 @@ DiscreteOrdinatesAdjointSolver::GetInputParameters()
 DiscreteOrdinatesAdjointSolver::DiscreteOrdinatesAdjointSolver(const InputParameters& params)
   : lbs::DiscreteOrdinatesSolver(params)
 {
-  basic_options_.AddOption<std::string>("REFERENCE_RF", std::string());
 }
 
 DiscreteOrdinatesAdjointSolver::DiscreteOrdinatesAdjointSolver(const std::string& solver_name)
   : lbs::DiscreteOrdinatesSolver(solver_name)
 {
-  basic_options_.AddOption<std::string>("REFERENCE_RF", std::string());
-}
-
-const std::vector<lbs::DiscreteOrdinatesAdjointSolver::RespFuncAndSubs>&
-DiscreteOrdinatesAdjointSolver::GetResponseFunctions() const
-{
-  return response_functions_;
 }
 
 void
 DiscreteOrdinatesAdjointSolver::Initialize()
 {
   LBSSolver::Initialize();
 
-  MakeAdjointXSs();
-  InitQOIs();
+  // Create adjoint cross sections
+  std::map<int, std::shared_ptr<MultiGroupXS>> matid_to_adj_xs_map;
+  for (const auto& [matid, xs] : matid_to_xs_map_)
+    matid_to_adj_xs_map[matid] =
+      std::make_shared<AdjointMGXS>(*std::dynamic_pointer_cast<MultiGroupXS>(xs));
+  matid_to_xs_map_ = std::move(matid_to_adj_xs_map);
 
-  // Initialize source func
-  auto src_function = std::make_shared<AdjointSourceFunction>(*this);
+  for (const auto& cell : grid_ptr_->local_cells)
+    cell_transport_views_[cell.local_id_].ReassingXS(*matid_to_xs_map_[cell.material_id_]);
```

Comment:
what is Reassing ?

Reply:
That is a typo in a routine that has been around for years now, haha! It should be `ReassignXS`.